vars/main.yml

```diff
@@ -2,7 +2,7 @@
 # vars file for nessus
 
 # The name of the Nessus Debian package file
-version: "8.2.3"
+version: "8.7.1"
```

Comment:
You really want to be using `defaults` instead of `vars` for this.  See [here](https://codereviewvideos.com/course/ansible-tutorial/video/variable-precedence-where-to-put-your-role-vars), for instance.

I have only recently come to understand the difference between `defaults` and `vars`, so the use of `vars` here is probably my fault.  Mea culpa.

Reply:
That link was informative and I agree about using `defaults` instead.